tools/releaser/scripts/new-version.sh

```diff
@@ -17,16 +17,29 @@ if [ "$NEW_RESOURCE_VERSION" == "$SDK_RESOURCE_VERSION" ]; then
 	new_minor_version=$((SDK_MINOR_VERSION + 1))
 	# Update the SDK_VERSION
 	SDK_VERSION="${SDK_MAJOR_VERSION}.${new_minor_version}.0"
+
+	echo "Print breaking changes"	
+	# shellcheck source=/dev/null
+	source "$script_path/breaking-changes.sh"
+	if [ -n "$BREAKING_CHANGES" ]; then
+		echo "BREAKING CHANGES DETECTED FOR NON MAJOR VERSION BUMP"
+		# shellcheck source=/dev/null
+		source "$script_path/update-version.sh"
+		exit 0;
+	fi
 else	 
 	# Update the SDK_VERSION
 	echo "Resource Version is not up to date. Changing major version."
-	NEW_MAJOR_VERSION="${NEW_RESOURCE_VERSION}001"
+	NEW_MAJOR_VERSION="v${NEW_RESOURCE_VERSION}001"
 	SDK_VERSION="v${NEW_MAJOR_VERSION}.0.0" 
+
```

Comment:
Looks like you have `v` before both versions, I think one should be omitted

```suggestion
	NEW_MAJOR_VERSION="v${NEW_RESOURCE_VERSION}001"
	SDK_VERSION="${NEW_MAJOR_VERSION}.0.0" 

```

Reply:
I see the comment is resolved but the repeated v is still present
Was I mistaken?
Looks like `SDK_VERSION` would evaluate to `vv2023-10-01001.0.0`

Reply:
I could swear I have resolved that comment 1h ago. It might be that commit were lost or my resolution were ignored. 
Sorry for that.